src/metabase/api/metric.clj

```diff
@@ -32,12 +32,12 @@
   ;; TODO - why can't set the other properties like `show_in_getting_started` when you create a Metric?
   (api/create-check Metric body)
   (let [metric (api/check-500
-                (db/insert! Metric
-                  :table_id    table_id
-                  :creator_id  api/*current-user-id*
-                  :name        name
-                  :description description
-                  :definition  definition))]
+                 (first (t2/insert-returning-instances! Metric
+                                                        :table_id    table_id
+                                                        :creator_id  api/*current-user-id*
+                                                        :name        name
+                                                        :description description
+                                                        :definition  definition)))]
```

Comment:
not great to lose the two-space indent, but…c'est la vie. People can fix it as they discover it if they care enough.

Reply:
these methods doesn't have `style/indent` since it supports multiple arity. so I'm not sure what is the correct way.